test/functional/admin/new_document_controller_test.rb

```diff
@@ -0,0 +1,54 @@
+require "test_helper"
+
+class Admin::NewDocumentControllerTest < ActionController::TestCase
+  setup do
+    login_as_preview_design_system_user :writer
+  end
+
+  view_test "GET #index renders the 'New Document' page with the header, all relevant radio selection options and inset text" do
+    get :index
+
+    assert_response :success
+    assert_select "h1.govuk-heading-xl", text: "New document"
+
+    radio_button_values.each do |value|
+      assert_select_radio_button(value)
+    end
+
+    assert_select ".govuk-inset-text", text: "Check the content types guidance if you need more help in choosing a content type."
+  end
+
+  test "access to the New document index page is forbidden for users without design system permissions" do
+    login_as :writer
+    get :index
+    assert_response :forbidden
+  end
+
+  test "POST #new_document_options_redirect redirects to #call-for-evidence if search option passed is call-for-evidence" do
```

Comment:
Should we be testing all the links to make sure they all redirect as expected? Maybe use parameterized tests to avoid having to duplicate the same test multiple for each link.

Reply:
Sounds good - I've now setup a parametized test that ensures each radio buttons redirects the user to the correct path, thanks @mtaylorgds